js/main.ts

```diff
@@ -51,3 +52,5 @@ export default function denoMain(name?: string): void {
     replLoop();
   }
 }
+
+denoMain.triggerLoadEvent = triggerLoadEvent;
```

Comment:
I think we need to at some point collect all private calls (e.g. `window.workerMain`, `window.xevalMain`) and this to a centralized location to avoid confusing users with other public ones. Maybe putting into `window._$$deno` so that they look like `window._$$deno.triggerLoadEvent`? (debating about the naming)

Reply:
I was a little uncomfortable about using denoMain as a namespace. I agree that we should have something like what you suggest.